tests/Integration/Behaviour/Features/Scenario/Carrier/carrier_management.feature

```diff
@@ -175,26 +225,219 @@ Feature: Carrier management
       | active       | true                               |
       | delay[en-US] | Shipping delay new                 |
 
+Scenario: Partially editing carrier with handling shipping
+    When I create carrier "carrier1" with specified properties:
+      | name             | Carrier 1                          |
+      | grade            | 1                                  |
+      | trackingUrl      | http://example.com/track.php?num=@ |
+      | position         | 2                                  |
+      | active           | true                               |
+      | delay[en-US]     | Shipping delay                     |
+      | shippingHandling | false                              |
+      | isFree           | false                              |
+      | shippingMethod   | 1                                  |
+      | taxRuleGroup     | US-AL Rate (4%)                    |
+      | rangeBehavior    | 1                                  |
```

Comment:
Same suggestion for range behaviour, in the behat features you should use understandable strings